src/Tribe/Assets.php

```diff
@@ -710,9 +702,25 @@ public function register( $origin, $slug, $file, $deps = [], $action = null, $ar
 
 		// Clean these
 		$asset->priority  = absint( $asset->priority );
-		$asset->in_footer = is_array( $asset->in_footer ) ? $asset->in_footer : (bool) $asset->in_footer; // Since WordPress 6.3, this parameter accepts an array argument.
 		$asset->media     = esc_attr( $asset->media );
 
+		// Since WordPress 6.3, the `in_footer` parameter accepts an array argument.
+		if ( version_compare( strtok( $wp_version, '-' ), '6.3', '>=' ) ) {
+			// if `in_footer` is set to boolean true, add it to the `in_footer` array. i.e. [ 'in_footer' => true ].
+			if ( (bool) $asset->in_footer ) {
+				$asset->in_footer['in_footer'] = (bool) $asset->in_footer;
```

Comment:
```
				$asset->in_footer['in_footer'] = (bool) $asset->in_footer;
```
This comes in as a `bool` and gets converted to an `array`?

Reply:
Yes,. instead of overloading the `in_footer` param where it's sent, we continue to send it as a bool, and if conditions are right (proper WP version and async of defer set) we convert it to an array that contains the original boolean value.

Reply:
Won't it `PHP Warning:  Uncaught Error: Cannot use a scalar value as an array` ?